README.md

```diff
@@ -1,18 +1,22 @@
 # REDI Sprint 2020 Frontend Development 2, Final Project
 
-Application Name: {app-name}
+Application Name: StyleList
 
-Student Name: {student-name}
+Student Name: Golovnev Sergei
 
-## {app-name}
+## StyleList
 
-{short explanation of your application}
+My product StyleList allows users to find their basic wardrobe key pieces based on their style preferences. User goes through a quiz where she is asked to choose between 2 outfits, then the product picks 4 most frequently liked outfits and shown it to user. Then user clicks on them and the product provides list of items used on the pictures.
```

Comment:
Only 'she' can go though the quiz?)) 
Better use just user (or they https://en.wikipedia.org/wiki/Singular_they)

Reply:
Of course, thank you!